docs/components/popover.md

```diff
@@ -1,760 +1,515 @@
-      I am placed on the {{ selectedPosition }}!
-    </template>
-  </KPop>
-</template>
-
-<script lang="ts">
-import { defineComponent } from 'vue'
-
-export default defineComponent({
-  data () {
-    return {
-      selectedPosition: 'auto',
-      positions: [
-        'auto',
-        'top',
-        'topStart',
-        'topEnd',
-        'left',
-        'leftStart',
-        'leftEnd',
-        'right',
-        'rightStart',
-        'rightEnd',
-        'bottom',
-        'bottomStart',
-        'bottomEnd'
-      ]
-    }
-  }
-})
-</script>
 ```
 
 ### positionFixed
 
-Use fixed positioning of the popover to avoid content being clipped by parental boundaries - defaults to `false`.
+A flag to use fixed positioning of the popover to avoid content being clipped by parental boundaries. Defaults to `true`.
 
-<div style="width: 300px; height: 125px; position: relative; overflow: hidden; z-index: 1; background-color: salmon;">
-  <KPop
-    title="Look Mah!"
-    width="170"
-    placement="right"
-  >
-    <KButton>Click</KButton>
+<div class="position-fixed-container">
+  <KPop button-text="Fixed positioning" positioning="top">
     <template #content>
-      My parent is too small 😭
+      Popover content.
     </template>
   </KPop>
-</div>
 
-```html
-<div style="width: 300px; height: 125px; position: relative; overflow: hidden; z-index: 1; background-color: salmon;">
-  <KPop
-    title="Look Mah!"
-    width="170"
-    placement="right"
-  >
-    <KButton>Click</KButton>
+  <KPop :position-fixed="false" positioning="top">
+    <KButton appearance="danger">
+      Non-fixed positioning
+    </KButton>
     <template #content>
-      My parent is too small 😭
-    </template>
-  </KPop>
-</div>
-```
-
-<div style="width: 300px; height: 125px; position: relative; overflow: hidden; z-index: 1; background-color: lightblue;">
-  <KPop
-    title="Look Mah!"
-    width="170"
-    placement="right"
-    :position-fixed="true"
-  >
-    <KButton>Click</KButton>
-    <template #content>
-      My parent is too small, but I don't care 😎
+      Popover content.
     </template>
   </KPop>
 </div>
 
 ```html
-<div style="width: 300px; height: 125px; position: relative; overflow: hidden; z-index: 1; background-color: lightblue;">
-  <KPop
-    title="Look Mah!"
-    width="170"
-    placement="right"
-    :position-fixed="true"
-  >
-    <KButton>Click</KButton>
-    <template #content>
-      My parent is too small, but I don't care 😎
-    </template>
-  </KPop>
-</div>
-```
-
-### width
-
-The width of the popover body - by default it is `200px`. Currently we support numbers (will be converted to `px`), `auto`, and percentages for width.
-
-<KPop title="Cool header" width="300">
-  <KButton>button</KButton>
+<KPop
+  :position-fixed="false"
+  button-text="Open popover"
+>
   <template #content>
-    I am 300 pixels wide!
+    Popover content.
   </template>
 </KPop>
+```
+
+### hideCloseIcon
+
+Boolean to hide close button in popover content.
```

Comment:
Is it intended that the content doesn't take the full width when possible? The close button seems to take the full height.

![image](https://github.com/Kong/kongponents/assets/67973710/41790163-cf1b-4124-b9eb-256d0297b22a)

Reply:
Yes it is intended. We can't make content overflow the close icon in any way so they display side-by-side. This looks better when title is present.